docs/scripts/generate-providers.js

```diff
@@ -1,13 +1,13 @@
-const path = require("path")
```

Comment:
Unnecessary file change, could you please revert?

Reply:
I executed `yarn lint:fix` for linting and it modified some of the files.